frontend/apps/filemanager/filemanagershortcuts.lua

```diff
@@ -186,32 +198,31 @@ function FileManagerShortcuts:deleteFolderShortcut(item)
         end
     end
     G_reader_settings:saveSetting("folder_shortcuts", folder_shortcuts)
-    self:init()
+    self:updateItemTable()
+end
+
+function FileManagerShortcuts:onSetDimensions(dimen)
+    self.dimen = dimen
 end
 
 function FileManagerShortcuts:onShowFolderShortcutsDialog()
-    local fm_bookmark = self:new{
+    self.fm_bookmark = Menu:new{
         title = _("Folder shortcuts"),
+        item_table = self.shortcuts,
```

Comment:
This `self.shortcuts` is not defined anywhere, so better use `item_table = nil` , as I guess you fill it later.

Reply:
Oh right, it's a leftover from a slightly earlier iteration.